src/SymbolCollector.Server/Program.cs

```diff
@@ -80,6 +80,10 @@ private static IHostBuilder CreateHostBuilder(string[] args) =>
                     o.AddExceptionFilterForType<OperationCanceledException>();
                     o.MinimumBreadcrumbLevel = LogLevel.Debug;
                     o.CaptureFailedRequests = true;
+
+                    // https://github.com/getsentry/symbol-collector/issues/205
+                    // o.CaptureBlockingCalls = true;
```

Comment:
We should get a lauchdarkly account and test this out with it (+ dogfood our feature flag integration)